tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1040,112 +1059,118 @@ def test_continuous_reads_during_pod_restart(self):
                                        restore_method=self.restore_method,
                                        restore_params={"deployment_name": self.deployment_name,
                                                        "deployment_backup":
-                                                           self.deployment_backup})
+                                                           self.deployment_backup},
+                                       clstr_status=True)
         LOGGER.debug("Response: %s", resp)
         assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
-        LOGGER.info("Step 7: Successfully started the pod")
+        LOGGER.info("Step 6: Successfully started the pod")
         self.restore_pod = False
-
-        LOGGER.info("Step 8: Check cluster status")
-        resp = self.ha_obj.poll_cluster_status(self.node_master_list[0], timeout=180)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 8: Cluster is in good state. All the services are up and running")
         event.clear()
         thread.join()
+        if CMN_CFG["dtm0_disabled"]:
+            thread1.join()
 
-        LOGGER.info("Verifying responses from background process")
-        responses = {}
+        LOGGER.info("Step 7: Verifying responses from background processes")
+        responses = dict()
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
         pass_logs = list(x[1] for x in responses["pass_res"])
         fail_logs = list(x[1] for x in responses["fail_res"])
-        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        pass_logs1 = fail_logs1 = list()
+        if CMN_CFG["dtm0_disabled"]:
+            responses1 = dict()
+            while len(responses1) != 2:
+                responses1 = output1.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+            pass_logs1 = list(x[1] for x in responses1["pass_res"])
+            fail_logs1 = list(x[1] for x in responses["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs + pass_logs1)
         assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
-        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs + fail_logs1)
         assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
-        LOGGER.info("Step 6: Successfully completed READs and verified DI on the written data in "
+        LOGGER.info("Step 7: Successfully completed READs and verified DI on the written data in "
                     "background")
 
-        LOGGER.info("Step 9: Create multiple buckets and run IOs")
-        self.test_prefix = 'test-34073-1'
+        LOGGER.info("Step 8: Perform WRITEs/READs-Verify")
+        if CMN_CFG["dtm0_disabled"]:
```

Comment:
For this step, we should call ha_s3_workload_operation in if else,
one with skipcleanup=False (DELETEs should happen)
other with True when dtm0 is False.

Reply:
Why deletes should happen? @ankita-93

Reply:
@ankita-93 for read/write test we can let go of deletes